stackClass.cpp

```diff
@@ -43,21 +43,33 @@ char pop(){
 }
 
 //Prints out stack values back to back (without spaces) without popping stack
-void display(){
+void stackClass::display(){
     if(topNode != NULL){
         stackClass* pointer = new stackClass;
         pointer = topNode;
-        while (pointer -> nextNode != NULL){
-            std::cout << pointer -> data;
+        // my compiler only prints until the second last element
+        // e.g if i pushed the string 'abcdef' it only prints 'fedcb'. so yeah...
+
+        // while (pointer -> nextNode != NULL){
+        //     std::cout << pointer -> data;
+        //     pointer = pointer -> nextNode;
+        // }

```

Comment:
This is weird.. the reason I added ' -> nextNode' (line 53) in the first place was because the compiler would print out the last element twice.
To fix this just remove the ' -> nextNode' in the while conditional instead.